js/smart-side-nav.js

```diff
@@ -0,0 +1,56 @@
+namespacer('bcpl');
+
+bcpl.smartSideNav = (($, window) => {
+	const navLinksSelector = '.secondary-nav nav ul li a';
```

Comment:
im not sure this is an appropriate time to fix, but based on your selector i would expect .secondary-nav to be class on a nav element, where as here is seems like its a container.

Reply:
The css class name could be changed to "secondary-nav-container", but I need the class there. Not worth changing right now.

Reply:
yea i figured as much